docker/test/integration/features/kubernetes.feature

```diff
@@ -0,0 +1,43 @@
+Feature: TailFile can collect logs from Kubernetes pods
```

Comment:
I see that we do not really add the apache license text in files in the docker directory, could this have unintended consequences?

Reply:
I'm not sure.  I have added a license header to new files containing code, but not to new files containing test data.  I think these feature files are test data.

Do you think we need to add a header to all files, or just that feature files are code and not test data, so should have a header?  There is no technical problem with adding headers to feature files; lines starting with # are ignored.

Reply:
I have added a license header to all new or modified `.py`/`.feature` files in https://github.com/apache/nifi-minifi-cpp/pull/1270/commits/b912ce6ca9ae4728c3db8dae7fc89814725d011f, and created a Jira to update the rest: [MINIFICPP-1764](https://issues.apache.org/jira/browse/MINIFICPP-1764)

Reply:
thanks!